ssg/oval.py

```diff
@@ -13,7 +13,7 @@
 from .rules import get_rule_dir_id, get_rule_dir_ovals, find_rule_dirs
 from .xml import ElementTree as ET
 from .xml import oval_generated_header
-from .yaml import process_file
+from .jinja import process_file_with_macros
```

Comment:
This is kinda funny. The import here was technically wrong: `ssg.yaml` imports `process_file` from `ssg.jinja`... 

Note that we don't have to import `process_file_with_macros` in `ssg.yaml` as sometimes we have two sets of logic; one with Jinja macros and one without (and use `process_file` in both).